controllers/admin/AdminCartsController.php

```diff
@@ -788,7 +788,18 @@ public function displayAjaxSearchCarts()
 
     public function ajaxReturnVars()
     {
-        $id_cart = (int) $this->context->cart->id;
+        if ((int) Tools::getValue('id_cart')) {
+            $this->context->cookie->id_cart = $id_cart = (int) Tools::getValue('id_cart');
+        } elseif (!$this->context->cookie->id_cart) {
+            $cart = new Cart();
+            $cart->id_currency = $this->context->currency->id;
+            $cart->id_lang = $this->context->language->id;
+            $cart->save();
+            $this->context->cookie->id_cart = $cart->id;
```

Comment:
```suggestion
            $this->context->cookie->id_cart = $id_cart = (int) $cart->id;
```

Reply:
I had it like this, but I thought it didn't comply with the code standards. Thanks.